examples/vision/md/temporal_latent_bottleneck.md

```diff
@@ -925,6 +981,12 @@ index = random.randint(0, config["batch_size"])
 orig_image = images[index]
 overlay_image = upsampled_heat_map[index, ..., 0]
 
+if keras.backend.backend() == "torch":
```

Comment:
Please note, if you use `ops.convert_to_numpy()` you don't need any torch specific code. This is not a blocker so we can merge regardless.